package.json

```diff
@@ -142,9 +142,11 @@
     "@material-ui/lab": "^4.0.0-alpha.60",
     "apexcharts": "3.32.1",
     "i18next-http-backend": "^1.4.1",
+    "market-sdk": "^2.1.0",
```

Comment:
Why is market XYZ here

Reply:
I think we don't need to remove the market sdk package.